src/main/java/com/example/model/Book.java

```diff
@@ -1,13 +1,27 @@
 package com.example.model;
 
+import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
 
 @Entity
+@Data
```

Comment:
```suggestion
@Getter
@Setter
@Table(.....
```

Reply:
Why using of @Getter and @Setter are better than @Data?

Reply:
https://medium.com/@miguelangelperezdiaz444/the-hidden-dangers-of-lombok-annotations-in-your-java-code-what-you-need-to-know-8acdce2d6b89